var/spack/repos/builtin/packages/ninja-fortran/package.py

```diff
@@ -4,10 +4,10 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.package import *
-from spack.util.executable import which_string
+from spack.pkg.builtin.ninja import NinjaBase
 
 
-class NinjaFortran(Package):
```

Comment:
This could probably be renamed to `NinjaKitware` since the Fortran part of this is upstreamed 1.10

Reply:
Or `NinjaJobsever` if we want the package name to reflect the changes to the base package.

Reply:
That works too. Although if there is ever another feature we want in Ninja that isn't up-streamed for a long time like module support or job server it would be nice to not have to change the name.

Reply:
Agreed, it could be renamed.  I had left it so far to avoid breaking existing usage, but we could pretty easily transition it.

Reply:
The `ninja -> ninja-build` is already breaking things, might as well rename now.
FWIW I now agree with Ryan's name suggestion, disregard mine.